dispatchers/job.rb

```diff
@@ -0,0 +1,36 @@
+module Dispatchers
+  class Job
+    JOBS = YAML.load_file('./data/jobs.yml')
+
+    attr_reader :bot
+
+    def initialize(bot)
+      @bot = bot
+    end
+
+    def more(callback)
+      data = JSON.parse(callback.data)
+      job = JOBS.detect { |j| j['id'] == data['job_id'] }
+      text = "*#{job['title']}*\nCompany: #{job['company']}\nLocation: #{job['location']}\n#{job['announce']}\n#{job['full_description']}"
+      bot.api.edit_message_text(
+        message_id: callback.message.message_id,
+        chat_id: callback.message.chat.id,
+        text: text,
+        parse_mode: :markdown
+      )
+    end
+
+    def jobs(ctx)
+      JOBS.each_with_index do |job, i|
```

Comment:
We actually can split data/messages/extended messages preparation from transport API. WDYT?